examples/externalContactsDemo/script.js

```diff
@@ -105,7 +109,7 @@ document.addEventListener('DOMContentLoaded', function () {
                         }
                     }
                 }).catch(err => {
-                    setErrorState(err);
+                    setErrorState(err.message);
```

Comment:
was this failing or just something you spotted?  Seems like the right change to make but tough to know in js and a catch block where err is `any`.

Dont mind the change ... just curious.

Reply:
When I tested the sample and got the permission error (missing app scope), the message shown in the progress bar was `[object Object]`, so I just made the change to show the real message sent by API.